```
/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/branch-master/swift/tools/swift-api-digester/swift-api-digester.cpp:1099:28: error: no member named 'Deinit' in 'swift::DeclBaseName::Kind'
  case DeclBaseName::Kind::Deinit:
       ~~~~~~~~~~~~~~~~~~~~^
/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/branch-master/swift/tools/swift-api-digester/swift-api-digester.cpp:1096:16: error: enumeration value 'Destructor' not handled in switch [-Werror,-Wswitch]
  switch (name.getKind()) {
               ^
/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/branch-master/swift/tools/swift-api-digester/swift-api-digester.cpp:1096:16: note: add missing switch cases
  switch (name.getKind()) {
               ^
2 errors generated.
```

--- a/tools/swift-api-digester/swift-api-digester.cpp
+++ b/tools/swift-api-digester/swift-api-digester.cpp
@@ -1096,9 +1096,12 @@ static StringRef getEscapedName(DeclBaseName name) {
   switch (name.getKind()) {
   case DeclBaseName::Kind::Subscript:
     return "subscript";
+  case DeclBaseName::Kind::Destructor:
+    return "deinit";
   case DeclBaseName::Kind::Normal:
     return llvm::StringSwitch<StringRef>(name.getIdentifier().str())
         .Case("subscript", "`subscript`")
+        .Case("deinit", "`deinit`")
         .Default(name.getIdentifier().str());
   }
 }

